### PR TITLE
feat: authentication pages

### LINKED ISSUE #5: Authentication pages
### Authentication pages
- [x] Login page
- [x] Signup page
- [x] Logout page

### PR DESCRIPTION
**Pull request by siddhigate**
### Authentication pages
- Login page
- Signup page
- Logout page

**Live Preview:** https://fashionly-ui-git-authentication-pages-siddhigate.vercel.app/

image
image
image

### CODE REVIEW

File: css/pages/authentication.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  font-size: 1.25rem;
+}
+
+.card-logout .link {
+  text-decoration: none;
+  border-bottom: 1px solid var(--clr-primary);
+}
+
+.card-logout .link:hover,
+.card-logout .link:hover {
```
Comment (Sen-442b) [inline css/pages/authentication.css:57]: Any specific reason to use the pseudo element on the same class twice?

**Review by Sen-442b — APPROVED** [review]
Looks Good to me

**Comment by preethasri** [comment]
good to go 👍

File: css/pages/authentication.css (hunk 1 above)
Comment (siddhigate) [reply css/pages/authentication.css:57]: Thanks for pointing it out! It was by mistake. Made the change

**Review by ZuberDunge — APPROVED** [review]
Well structured code, UI looks pretty, good work Siddhi. Good to go from my side.

**Pull request merged**